Story/StoryHooks.cs

```diff
@@ -612,37 +612,9 @@ private void KarmaLadderScreen_Singal(On.Menu.KarmaLadderScreen.orig_Singal orig
 
         private void Player_Update(On.Player.orig_Update orig, Player self, bool eu)
         {
+            if (isStoryMode(out var gameMode) && OnlinePhysicalObject.map.TryGetValue(self.abstractCreature, out var oe) && oe.isMine)
+                gameMode.storyClientSettings.readyForWin = false;
```

Comment:
would we have issues where player_update would update our local readyForWin to false and DenClose would update to true back and forth?

Reply:
this back and forth happens within player.update, readyForWin should be consistent outside of this function